common/src/test/java/com/revenuecat/purchases/common/offlineentitlements/OfflineCustomerInfoCalculatorTest.kt

```diff
@@ -0,0 +1,210 @@
+package com.revenuecat.purchases.common.offlineentitlements
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.ibm.icu.impl.Assert.fail
+import com.revenuecat.purchases.CustomerInfo
+import com.revenuecat.purchases.OwnershipType
+import com.revenuecat.purchases.PeriodType
+import com.revenuecat.purchases.Store
+import com.revenuecat.purchases.common.AppConfig
+import com.revenuecat.purchases.common.DateProvider
+import com.revenuecat.purchases.common.ago
+import com.revenuecat.purchases.common.fromNow
+import com.revenuecat.purchases.utils.stubStoreTransactionFromPurchaseHistoryRecord
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.annotation.Config
+import java.util.Date
+import org.assertj.core.api.Assertions.assertThat
+import org.json.JSONObject
+import kotlin.time.Duration.Companion.days
+import kotlin.time.Duration.Companion.hours
+
+@RunWith(AndroidJUnit4::class)
+@Config(manifest = Config.NONE)
+class OfflineCustomerInfoCalculatorTest {
+    private val testDate = 1.hours.ago()
+    private val testDatePlusOneDay = 1.days.fromNow()
```

Comment:
This is technically `testDate + 1 day + 1 hour`, not `testDate + 1 day`.

Reply:
oh good point. I am going to rename to just future date and past date